src/main/scala/io/radicalbit/nsdb/connector/kafka/sink/NSDbSinkWriter.scala

```diff
@@ -148,14 +153,41 @@ class NSDbSinkWriter(connection: NSDB,
 
 object NSDbSinkWriter {
 
-  import io.radicalbit.nsdb.connector.kafka.sink.conf.Constants._
-
   private val logger = Logger(LoggerFactory.getLogger(classOf[NSDbSinkWriter]))
 
   private val defaultTimestampKeywords = Set("now", "now()", "sys_time", "sys_time()", "current_time", "current_time()")
 
   private def getFieldName(parent: Option[String], field: String) = parent.map(p => s"$p.$field").getOrElse(field)
 
+  private def policy(maxRetries: Int) = RetryPolicies.limitRetries(maxRetries - 1)
+
+  private def onFailure(throwable: Throwable, details: RetryDetails): Future[Unit] = {
+    logger.debug(throwable.getMessage)
+    Sleep[Future].sleep(1000.milliseconds)
```

Comment:
could we define this sleep interval as a config value with 1 second as the default value?